collector/README.md

```diff
@@ -1,25 +1,48 @@
-# Rust Compiler Performance Benchmarking
+# Rust Compiler Performance Benchmarking and Profiling
 
 ## How to build
 
+Before doing anything else, you should build `collector` (for running the
+benchmarks) and `site` (for viewing the results):
 ```
-cargo build -p collector --release
+cargo +nightly build --release
 ```
 
-## Benchmarking committed builds
+## Benchmarking
 
-To benchmark builds from a rustc repository:
-```
-./target/release/collector --output-repo $RUSTC_TIMING process
-```
+This section is about benchmarking rustc, i.e. measuring its performance on the
+standard benchmark suite. The most likely reason for doing this is to evaluate
+the performance effect of a change you've made to rustc. It's also done
+regularly by the benchmark server.
 
-`$RUSTC_TIMING` is a path (relative or absolute) to a clone of the
-`https://github.com/rust-lang-nursery/rustc-timing` repository, in which the
-output data will be placed and committed.
+### How to benchmark a change using the benchmark server
 
-## Benchmarking local builds
+An easy (but slow) way to benchmark the performance effect of a change is to
+request a run on the benchmark server for a specific PR.
 
-To benchmark local builds:
+First, create a PR with the changes.
+
+After that, you need try privileges, or the assistance of someone with try
+privileges. Ping `simulacrum` on IRC as a starting point.
+
+That person will enter `@bors try` as a comment in the PR. This queues a normal
+try build on Travis, which takes some time.
+
+Once the try build has completed, that person will enter `@rust-timer build
+$MERGE` as a comment in the PR, where `$MERGE` is the full 40 character merge
+revision ID from the try build. This queues a benchmarking run, which also
+takes some time. Upon completion, a URL is posted in the PR at which the
+results can be viewed.
```

Comment:
Might be good to clarify here that the URL is posted instantly; results are generally available in ~6 hours or so I think